free-1.md

```diff
@@ -10,34 +10,45 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I think the need being met here is and easy and user-friendly way to manipulate sounds and frequencies that the user records. As of now, I believe the only way that users can do what I'd like to have happen is by recording a sound, then using FL or other studio-type software to pitch bend to an appropriate frequency.  I don't believe there is an easy way to record and manipulate sounds in the way that I envision it.
+
+I imagine the experience as follows: A user records themselves singing or playing an instrument. They then use the DSL in order to add thirds, fourths, and other intervals as they choose. They can also do other simpler things such as slowing down and speeding up 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think a DSL is appropriate because it would allow for lots of different types of behaviours from the user, and would also allow them lots of precision. For example, if a user wants an octave above a recorded sound at 50 percent volume with .1second reverb, that would be easy to do and straightforward to write. I think it addresses the need by making it really intuitive to do.
```

Comment:
As I'm understanding this now, it seems like you're envisioning this as a more complete comprehensive version of the lab we did in class. Seems good so far!